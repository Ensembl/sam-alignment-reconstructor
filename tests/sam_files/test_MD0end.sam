MS2007487-600V3:1:2109:08999:01136	163	cecr1a_T2_25_17051701-17051898	1	70	99M5I99M	=	1	197	TTCGAAATCTCACGCTCTTTACTGAAGACCCAGNTAGAGCTTATCCTAATCAGGATACTGTGTGGGANAGNTTCGAGCAGGTATTTTTAGTGNCCTATGCCCCACTATGGTCACTTATGCCCCTGTCTTTAAAGACTACCTCTATGAAGGTCTCCGACAGTTTTATGAGGACAACATCACGTATGTGGAAATCAGAGCACTNN	GGGGFFFGGGGGGGGGGGGGGGGGGGGGGGGGC#:CDEFGGGGGGGGFGGGGGGGGGGGGGGGGGG8#9B#:AFGGGGGGFGFDFGGGGGGG#9ABEEFFFGGEEFGGGGGGGGFFFGGGGGGGGGFGGGCFGGGGGGGGFFCGGGGGGGGGGGFCCEEGGGFGGGGGG6CCFGC9?F+@CDDCFGFAFFC>>CFF>AAA9##	PG:Z:novoalign	AS:i:242	UQ:i:242	NM:i:16	MD:Z:33A33G2G21G6G0G0T72T9G11G0C0	PQ:i:448	SM:i:70	AM:i:70	BC:Z:ATCCCG
