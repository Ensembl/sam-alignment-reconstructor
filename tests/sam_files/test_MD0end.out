MS2007487-600V3:1:2109:08999:01136
TTCGAAATCTCACGCTCTTTACTGAAGACCCAGATAGAGCTTATCCTAAT
|||||||||||||||||||||||||||||||||:||||||||||||||||
TTCGAAATCTCACGCTCTTTACTGAAGACCCAGNTAGAGCTTATCCTAAT

CAGGATACTGTGTGGGAGAGGTTCGAGCAGGTATTTTTAGTGGCCTATG-
|||||||||||||||||:||:|||||||||||||||||||||:|||||| 
CAGGATACTGTGTGGGANAGNTTCGAGCAGGTATTTTTAGTGNCCTATGC

----GGTTGGTCACTTATGCCCCTGTCTTTAAAGACTACCTCTATGAAGG
    :::|||||||||||||||||||||||||||||||||||||||||||
CCCACTATGGTCACTTATGCCCCTGTCTTTAAAGACTACCTCTATGAAGG

TCTCCGACAGTTTTATGAGGACAACATCATGTATGTGGAGATCAGAGCAC
|||||||||||||||||||||||||||||:|||||||||:||||||||||
TCTCCGACAGTTTTATGAGGACAACATCACGTATGTGGAAATCAGAGCAC

TGC
|::
TNN

